client/app/bundles/course/assessment/pages/AssessmentShow/QuestionsManager.tsx

```diff
@@ -66,14 +68,17 @@ const QuestionsManager = (props: QuestionsManagerProps): JSX.Element => {
     if (sourceIndex === destinationIndex) return;
 
     moveItemAndUpdate(sourceIndex, destinationIndex);
+    props.setSyncStatus('Syncing');
```

Comment:
Similarly, find other places where you're using the constants `KODITSU_SYNC_STATUS` and update.

Reply:
done